freesound/settings.py

```diff
@@ -171,6 +171,17 @@
 MAX_UNMODERATED_SOUNDS_IN_HOME_PAGE = 5
 ALLOWED_AUDIOFILE_EXTENSIONS = ['wav', 'aiff', 'aif', 'ogg', 'flac', 'mp3']
 
+# After download modal
+AFTER_DOWNLOAD_MODAL = 'survey'  # Set it to none for no modal after download
```

Comment:
This kind of calls for an ENUM or pre-defined constants...

Magic strings nooooh!